db/migrate/20221022093453_add_reference_to_tasks.rb

```diff
@@ -0,0 +1,6 @@
+class AddReferenceToTasks < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :tasks, :project_id
+    add_reference :tasks, :project, foreign_key: true
```

Comment:
Not in new migration! You just added migration in this PR, it's not in the project yet. Therefore, you should change the existing one, and not produce new ones.